Userland/more.cpp

```
@@ -36,7 +36,7 @@ static void wait_for_key()
     printf("\033[7m--[ more ]--\033[0m");
     fflush(stdout);
     char dummy;
-    read(key_fd, &dummy, 1);
+    (void)read(key_fd, &dummy, 1);
     printf("\n");
```

That is 😟 though

We shouldn't ignore errors, not EOF.